charts/shoot-core/charts/vpn-shoot/templates/deployment.yaml

```diff
@@ -70,11 +70,11 @@ spec:
             - NET_ADMIN
         resources:
           requests:
-            cpu: 50m
-            memory: 50Mi
+            cpu: 500m
+            memory: 500Mi
```

Comment:
Can we lower this? This seems to be way too much. Having high limits is fine, but high requests is in 99% cases not needed.

Reply:
Okay, will keep the limits high as discussed.